endpoints/put.go

```diff
@@ -121,6 +121,11 @@ func parsePutObject(p putObject) (string, error) {
 		return "", utils.NewPBCError(utils.NEGATIVE_TTL, fmt.Sprintf("ttlseconds must not be negative %d.", p.TTLSeconds))
 	}
 
+	// Make sure data type is specified
+	//if len(p.Type) == 0 {
+	//	return "", utils.NewPBCError(utils.MISSING_TYPE)
+	//}
+
```

Comment:
Could this commented out code be deleted?

Reply:
Removed. Also corrected the `%v` string formatter in line 141 below:
```
140       } else {
141 -         return "", utils.NewPBCError(utils.UNSUPPORTED_DATA_TO_STORE, fmt.Sprintf("Type must be one of [\"json\", \"xml\"]. Found %v", p.Type))
    -         return "", utils.NewPBCError(utils.UNSUPPORTED_DATA_TO_STORE, fmt.Sprintf("Type must be one of [\"json\", \"xml\"]. Found '%s'", p.Type))
142       }
143  
144       return toCache, nil
145   }
```